sdk/core/azure-core/test/ut/logging.cpp

```diff
@@ -1,226 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
nit: leave this test file name as is.

this will also make it easier to review any test changes.